packages/shortest/src/index.ts

```diff
@@ -121,6 +121,7 @@ function createTestChain(
     const tests = nameOrFn.map((name) => {
       const test: TestFunction = {
         name,
+        filePath: "",
```

Comment:
There is no access to `TestFunction` here, and this is not really used yet in this PR.